source/shared/core_sqlsrv.h

```diff
@@ -56,10 +56,9 @@
 
 // #define MultiByteToWideChar SystemLocale::ToUtf16
 
-
-
 #define stricmp strcasecmp
 #define strnicmp strncasecmp
+#define strnlen_s(s) strnlen_s(s, INT_MAX)
```

Comment:
There are two definitions for strnlen_s? Why not put this line in Stringfunctions.h? Then you should be able to remove the definition from StringFunctions.cpp.

Reply:
No this is just a macro, mainly for WIN32, because strnlen_s() is builtin. Besides, StringFunctions.h is not included in Windows build

Reply:
Ok I see.

Reply:
btw have you checked that this compiles on all platforms? I'm just a bit concerned because strnlen_s is part of the c11 standard, not c++11.

Reply:
yes, I've run customized in Test Lab last Friday